altair/vegalite/v5/schema/_config.py

```diff
@@ -0,0 +1,1055 @@
+# The contents of this file are automatically written by
+# tools/generate_schema_wrapper.py. Do not modify directly.
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Literal, Sequence, TypedDict
+
+# ruff: noqa: F405
+if TYPE_CHECKING:
+    from ._typing import *  # noqa: F403
+    from .core import *  # noqa: F403
+    from .core import Dict
+
+
+class RectConfigKwds(TypedDict, total=False):
+    """Placeholder doc."""
+
+    align: Align_T
+    angle: float
+    aria: bool
+    ariaRole: str
+    ariaRoleDescription: str
+    aspect: bool
+    baseline: str | Baseline_T
+    binSpacing: float
+    blend: Blend_T
+    color: str | LinearGradient | RadialGradient | ColorName_T
+    continuousBandSize: float
+    cornerRadius: float
+    cornerRadiusBottomLeft: float
+    cornerRadiusBottomRight: float
+    cornerRadiusTopLeft: float
+    cornerRadiusTopRight: float
+    cursor: Cursor_T
+    description: str
+    dir: TextDirection_T
+    discreteBandSize: float | RelativeBandSize
+    dx: float
+    dy: float
+    ellipsis: str
+    endAngle: float
+    fill: str | None | LinearGradient | RadialGradient | ColorName_T
+    fillOpacity: float
+    filled: bool
+    font: str
+    fontSize: float
+    fontStyle: str | FontStyle
```

Comment:
TODO: `FontStyle`

Reply:
Turns out this is a direct alias for `str`, with no subclasses or properties.
Seems like a good rule for `use_concrete=True` to skip `SchemaInfo.title` if `SchemaInfo` meets criteria like this.

https://github.com/vega/altair/blob/cfab6601aa649227c20880d05ae27799297694c3/altair/vegalite/v5/schema/vega-lite-schema.json#L10983-L10985

Reply:
Fixed in: 
- [`ecf7fd5` (#3536)](https://github.com/vega/altair/pull/3536/commits/ecf7fd5f1dd45a68083c62b731f893c95e073519)
- [`f4cb0d4` (#3536)](https://github.com/vega/altair/pull/3536/commits/f4cb0d44aa7bfa7f3738472782a41edaaab6924b)